components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIDefinition.java

```diff
@@ -163,24 +162,28 @@ public abstract String populateCustomManagementInfo(String oasDefinition, Swagge
      *
      * @param api            API
      * @param oasDefinition  OAS definition
-     * @param hostsWithSchemes host addresses with protocol mapping
+     * @param tenantDomainOrOrganization The tenant domain or the organization
+     * @param environmentName The name of the environment
      * @return updated OAS definition
      * @throws APIManagementException throws if an error occurred
      */
-    public abstract String getOASDefinitionForStore(API api, String oasDefinition, Map<String, String> hostsWithSchemes)
+    public abstract String getOASDefinitionForStore(API api, String oasDefinition,
+                                                    String tenantDomainOrOrganization, String environmentName)
```

Comment:
Shall we rename all the places of `tenantDomainOrOrganization` as `organization`
Because for `organization` we can set tenantDomain as we as our usual implementation and its already implied.

```suggestion
                                                    String tenantDomainOrOrganization, String environmentName)
```

Reply:
Resolved in [c436bec](https://github.com/wso2/carbon-apimgt/pull/11430/commits/c436becef3f19bb2e780adbfe026acfac7b16431)